projects/plugins/boost/app/modules/cache/Boost_Cache.php

```diff
@@ -254,4 +272,208 @@ public function is_backend() {
 
 		return $is_backend;
 	}
+
+	/*
+	 * Checks if the post type is public.
+	 *
+	 * @param WP_Post $post - The post to check.
+	 * @return bool - True if the post type is public.
+	 */
+	protected function is_visible_post_type( $post ) {
+		$post_type = is_object( $post ) ? get_post_type_object( $post->post_type ) : null;
+		if ( empty( $post_type ) || ! $post_type->public ) {
+			return false;
+		}
+		return true;
+	}
+
+	/*
+	 * Delete the cache for the front page and paged archives.
+	 * This is called when a post is edited, deleted, or published.
```

Comment:
```suggestion
	 * Delete the cache for the front page if it will be affected by changes to the specified post.
	 * This is called when a post is edited, deleted, or published.
```

Reply:
I decided to remove the code that checked if the post was on the front page because invalidate_home_page() now deletes the paged archives now, so the comment is correct.